Cargo.toml

```diff
@@ -6,14 +6,18 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-argh = { version = "0.1.4", default-features = false }
```

Comment:
I tend to prefer argh as it's lighter weight, but if using clap makes the experience markedly better for people then I don't have much objection.

Reply:
Yeah, this was definitely an opinionated change. What you get with clap is stronger consistency with cargo, and how arguments are parsed and interpreted, access to libs that tie into clap (e.g. clap-verbosity-flag, completions, mangen, color in the help output, etc. It's little things like how the option alignment looks too. Your call on this obviously - as mentioned this PR was more about communicating ideas than about saying hey merge this please.